yrs/src/block_store.rs

```diff
@@ -307,52 +305,30 @@ impl ClientBlockList {
     /// squashed into its left neighbor. In such case a squash result will be returned in order to
     /// later on rewire left/right neighbor changes that may have occurred as a result of squashing
     /// and block removal.
-    pub(crate) fn squash_left(&mut self, index: usize) -> Option<SquashResult> {
```

Comment:
Removed squash result - now GC compaction happens immediately during squashing, which was triggering some of the errors in small-test-dataset.